football-manager/src/lib/Header/NotifyButton.svelte

```diff
@@ -0,0 +1,36 @@
+<script>
+  // This will be changed later when the backend is implemented
+  let notifications = ["test", "test2"];
+  let count = notifications.length;
```

Comment:
Changing variable name from count to counter

Reply:
This will change after we have implemented the backend.